src/status_im2/contexts/chat/bottom_sheet_composer/sub_view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.sub-view
+  (:require
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [react-native.linear-gradient :as linear-gradient]
+    [react-native.reanimated :as reanimated]
+    [status-im2.contexts.chat.bottom-sheet-composer.style :as style]))
+
+(defn bar
+  []
+  [rn/view {:style (style/bar-container)}
+   [rn/view {:style (style/bar)}]])
+
+(defn blur-view
+  [layout-height]
+  [:f>
```

Comment:
Same about functional components in this component and `gradients` in this namespace